Gruntfile.js

```diff
@@ -135,6 +137,31 @@ module.exports = function(grunt) {
 				}]
 			}
 		},
+
+		babel: {
+			options: {
+				sourceMap: false,
+				presets: [
+					'es2015',
+					'stage-2',
```

Comment:
I copied most of the transforms from the Calypso babel config, if we want to update the component to our coding style